systemtests/src/main/java/io/enmasse/systemtest/bases/iot/ITestIoTBase.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.bases.iot;
+
+import io.enmasse.systemtest.model.addressspace.AddressSpaceType;
+
+public interface ITestIoTBase {
```

Comment:
I think for future is better to let this be interface, we can add some methods with default value in future :)

Reply:
You can add methods for enums as well. And this is all about "addresses". So, how is this better?

Reply:
@ctron and what about Junit5 tags? how can you add tak on enum? it's imposible, and in time when we will create for example global iot tag we need to copy this tag on both intergaces instead of add this tag only on paren interface...